framework/auth/cas.py

```diff
@@ -170,6 +170,16 @@ def revoke_application_tokens(self, client_id, client_secret):
         else:
             self._handle_error(resp)
 
+    def revoke_personal_token(self, token_id):
```

Comment:
CAS doesn't share. There are two separate databases with minimal overlap. It will assume this kind of token is valid until it is told otherwise.